### ISSUE TITLE
convert vignettes to Rmd and create pkgdown site

### ISSUE THREAD

**Issue by timelyportfolio** [braverock/PerformanceAnalytics#85]
Based on recent activity, there appears to be an effort to update and submit a new version to CRAN.  If this is the case, I think some updated Rmd vignettes (possibly with [tint](https://github.com/eddelbuettel/tint)) might be nice.  I'll volunteer if there is interest.

**Comment by braverock** [comment]
Always happy to have more contributions and contributors.  What do you have in mind?

**Comment by timelyportfolio** [comment]
@braverock, I'll try to convert one of the vignettes as a starting point, and then we can go from there.  Thanks as always for your incredible work on R + finance.

**Comment by timelyportfolio** [comment]
@braverock, sorry this took so long, but I quickly (not complete converted) the PA-charts vignette to `Rmd` and ran `pkgdown::build_site().  http://timelyportfolio.github.io/PerformanceAnalytics/ is the result.  If you like it, then I am willing to try to convert the other vignettes

**Comment by braverock** [comment]
I like it!  what are the thoughts from @peterccarl and @joshuaulrich  ?

I'm not sure that there are a lot of reasons to spend too much time converting existing vignettes rather than writing new ones.  `knitr` can render Sweave vignettes too, it is more about changing the VignetteEngine, if I understand the process correctly.

**Comment by peterccarl** [comment]
I agree, it looks very nice.  Thank you for doing this!  pccOn Apr 8, 2018 11:20 AM, Joshua Ulrich <notifications@github.com> wrote:I think it looks great!  I do have a couple thoughts.  Neither of them are strong suggestions for anyone to actually implement.
Group functions by topic on the reference page.Indent or change the font size of the sub-topics in the vignette's contents navigation bar.

**Comment by timelyportfolio** [comment]
@braverock @joshuaulrich @peterccarl I'll take this as a reason to keep moving forward.  Styling is easy, but I wanted to be careful not to add another dependency.  I had originally done the vignette with [`tint]`(https://github.com/eddelbuettel/tint), but as far as I can tell vignettes require at least a `SUGGESTS`, so then I just chose the basic vignette `Rmd`.  In terms of the `pkgdown` site with `gh-pages`, we can easily add `HTML/CSS/JS` frameworks.

I have wanted for a long time to do a vignette on styling `PerformanceAnalytics` tables and charts.  Maybe that would make a good test case.

Converting existing vignettes is fairly time consuming, but I think eventually we would want all to be consistent.  

Thanks for all you work on this and everything else finance/time-series in `R`.
